assets/xml/objects/object_stk.xml

```diff
@@ -1,110 +1,138 @@
-        <Animation Name="object_stk_Anim_01C21C" Offset="0x1C21C" />
-        <Animation Name="object_stk_Anim_01D008" Offset="0x1D008" />
-        <Animation Name="object_stk_Anim_01D3D0" Offset="0x1D3D0" />
+        <Animation Name="gSkullKidLookAroundAnim" Offset="0x1030" /> <!-- Original name is "kyoro" (onomatopoeic for "looking around restlessly") -->
+        <Animation Name="gSkullKidCallDownMoonStartAnim" Offset="0x2774" /> <!-- Original name is "sk_moon_hikiyose" ("draw or pull the moon") -->
+        <Animation Name="gSkullKidCallDownMoonLoopAnim" Offset="0x3068" /> <!-- Original name is "sk_moon_hikiyose_loop" -->
+
+        <!-- DisplayLists for an unused Link Mask -->
+        <DList Name="gSkullKidLinkMask1DL" Offset="0x39A0" />
+        <DList Name="gSkullKidLinkMask2DL" Offset="0x46B0" />
+        <DList Name="gSkullKidLinkMask3DL" Offset="0x53C0" />
+
+        <!-- Majora's Mask Eyes DisplayList and Texture -->
+        <DList Name="gSkullKidMajorasMaskEyesDL" Offset="0x5870" />
+        <Texture Name="gSkullKidMajorasMaskEyeTex" OutName="skull_kid_majoras_mask_eye" Format="rgba16" Width="32" Height="32" Offset="0x5920" />
+
+        <!-- DisplayLists for Majora's Mask -->
+        <DList Name="gSkullKidMajorasMask1DL" Offset="0x6BB0" />
+        <DList Name="gSkullKidMajorasMask2DL" Offset="0x79F0" />
+
+        <!-- Unused Eyes DisplayList and Texture -->
+        <DList Name="gSkullKidUnusedEyesDL" Offset="0x7E10" />
+        <Texture Name="gSkullKidUnusedEyeTex" OutName="skull_kid_unused_eye" Format="rgba16" Width="8" Height="8" Offset="0x7EA0" />
+
+        <!-- Various Hand DisplayLists -->
+        <DList Name="gSkullKidUntexturedLeftHand" Offset="0x84C0" />
+        <DList Name="gSkullKidOpenLeftHand" Offset="0x87B0" />
+        <DList Name="gSkullKidFluteHoldingLeftHand" Offset="0x8A80" />
+        <DList Name="gSkullKidTwoFingersExtendedLeftHand" Offset="0x90C0" />
+        <DList Name="gSkullKidMaskHoldingRightHand" Offset="0x9710" />
+        <DList Name="gSkullKidUntexturedRightHand" Offset="0x9AC0" />
+        <DList Name="gSkullKidOcarinaHoldingRightHand" Offset="0x9DA0" />
+
+        <!-- Head DisplayLists with two different expressions -->
+        <DList Name="gSkullKidNormalEyesDL" Offset="0xA530" />
+        <DList Name="gSkullKidNormalHeadDL" Offset="0xA5C0" />
+        <DList Name="gSkullKidLaughingEyesDL" Offset="0xAE30" />
+        <DList Name="gSkullKidLaughingHeadDL" Offset="0xAEC0" />
+
+        <Animation Name="gSkullKidTelescopeLookUpStartAnim" Offset="0xBB2C" /> <!-- Original name is "stk_miageru" ("look up") -->
+        <Animation Name="gSkullKidTelescopeLookUpLoopAnim" Offset="0xC964" /> <!-- Original name is "stk_miageru_loop" -->
+        <DList Name="gSkullKidOcarinaOfTimeDL" Offset="0xCAD0" />
+        <Animation Name="gSkullKidTPoseAnim" Offset="0xCC94" /> <!-- Original name is "stk_pause" -->
```

Comment:
You mean the name of the animation? He's literally just doing a T-pose; I'm not sure how to describe it better than that:

![image](https://user-images.githubusercontent.com/12245827/172739856-41757bcb-b928-4d8b-b6f5-d18e3f72c758.png)

Reply:
Er wait, it has zero out of zero frames, is it possible this is another case z64Utils not handling animations correctly?

Reply:
That's just a UI glitch. The animation actually has 19 frames. Good to double-check that, though.

![image](https://user-images.githubusercontent.com/12245827/172740471-50cb4d07-ab40-48b6-b9a8-a4e9bab2843b.png)

Reply:
That does look rather odd... used?

Reply:
Used by unused actors, yes. You know those actors that have a description like "Skull Kid stuck in a T-pose"? Dm_Sa is one of them for example, and you can see it in this PR. This animation is why they T-pose.

Reply:
I suppose there's no reason to give it a better name, really, then.

Reply:
Yeah, I don't think it can be described any better than "it's a T-pose", nor is it really that important to do so, since no used actors actually do anything with it. I'm not sure why he even has this animation present, but oh well.